contract-tests/Cargo.toml

```diff
@@ -13,7 +13,7 @@ actix = { version = "0.13.1"}
 actix-web = { version = "4"}
 reqwest = { version = "0.11.6", default-features = false, features = ["json", "rustls-tls"] }
 env_logger = { version = "0.10.0" }
-hyper = { version = "0.14.17", features = ["client", "http1", "tcp"] }
+hyper = { version = "0.14.19", features = ["client", "http1", "tcp"] }
```

Comment:
This isn't the latest hyper 0.14.x version. This is the minimum required to fix the CVE. Figured leaving as much room in version matching as possible would help keep upstream packages smaller.